apps/trading/pages/portfolio/deposit/deposit-container.tsx

```diff
@@ -46,6 +48,7 @@ export const DepositContainer = ({
             requiredConfirmations={ethereumConfig.confirmations}
             assets={data.assets}
             initialAssetId={assetId}
+            isFaucetable={VEGA_ENV === 'MAINNET'}
```

Comment:
fix: (blocking): Can only be faucetable if the environment is not mainnet
```suggestion
            isFaucetable={VEGA_ENV !== 'MAINNET'}
```